packages/datetime/src/datePicker.tsx

```diff
@@ -43,6 +43,12 @@ export interface IDatePickerProps extends IDatePickerBaseProps, IProps {
      * The following props are managed by the component and cannot be configured:
      * `canChangeMonth`, `captionElement`, `fromMonth` (use `minDate`), `month` (use
      * `initialMonth`), `toMonth` (use `maxDate`).
+     *
+     * In case of supplying your owner `renderDay` function, make sure to provide the appropriate class to obtain default
```

Comment:
slight nitpick:
```suggestion
     * In case of supplying your owner `renderDay` function, make sure to apply the appropriate 
     * CSS wrapper class to obtain default Blueprint styling.
```